.github/workflows/ci-cd.yml

```
@@ -391,8 +402,14 @@ jobs:
 
   docker-build:
     name: Docker 이미지 빌드
-    needs: build
-    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.pull_request.merged == true)
+    # PR 머지 시에는 build를 기다리지 않고 바로 실행
+    needs: [lint, test, build]
+    if: |
+      github.event_name == 'push' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)